### DISCUSSION TITLE
Hidden organization

### DISCUSSION THREAD

**Discussion by ohassidi** [community/community#144998] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi I have an organization and I want to hide it, so it won't be available in search or by api calls like  https://api.github.com/orgs/{org} of course if I have the proper token it should work (what tokens should work here?)

Thanks

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @ohassidi!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the General category. I've gone ahead and moved it for you. Good luck!
